frontend/src/components/pages/TeamDashboard/utils.js

```diff
@@ -1,4 +1,4 @@
-export function updateActivityLogDayCounts({ activityLogDayCounts }) {
+export function updateActivityLogDayCounts({ activityLogDayCounts }, dateFilter) {
```

Comment:
Why use positional arguments? Isn't destructuring generally much safer? And it's good to be consistent. 

`export function updateActivityLogDayCounts({ activityLogDayCounts, dateFilter }) `